osgp/shared/osgp-ws-core/src/main/resources/schemas/devicemanagement-ws-core.xsd

```diff
@@ -183,8 +183,7 @@
   <xsd:element name="UpdateDeviceRequest">
     <xsd:complexType>
       <xsd:sequence>
-        <xsd:element name="DeviceIdentification" type="common:Identification" />
```

Comment:
I am surprised to see `DeviceIdentification` removed at this location in the XSD, after our conversation about my earlier review comments.

I thought things should be interpreted as follows:
`UpdateDeviceRequest`  ->  the request to update a device
  `DeviceIdentification`  ->  the identification of the device to update
  `UpdatedDevice`  ->  holder containing values to set while updating the device

Since the device identification should not be changed, I's expect it not to be among the elements of `UpdatedDevice` any longer.
Didn't we understand each other in our earlier chat, or did you come to a different understanding, leading to this change instead of what I expected?